### ISSUE TITLE
Logger configurations should be fixed

### ISSUE THREAD

**Issue by whitphx** [aiortc/aiortc#446]
Hello, thank you for the great work!

I think the current configurations of loggers in `aiortc` have 2 problems:

1. The loggers do not have a common prefix like `aiortc.`, then library users cannot have access to the loggers, for example to inject handlers, easily.
    * For example, `aiortc.rtcdatachannel` module now has a logger named as `datachannel` as below, however it should be at least like `aiortc.datachannel`. I prefer the same name to the module itself, `aiortc.rtcdatachannel`, for simplicity, which can simply be achieved by `logging.getLogger(__name__)` (Please see [this document](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial)).
      https://github.com/aiortc/aiortc/blob/15d8fa411ef0bfd29f18a81c0d5032fc2563add4/src/aiortc/rtcdatachannel.py#L9 
2. The loggers have non-null handlers by default, then they output logs to stdout without intensions of library users. When used as a library, the package should not output any logs by their own, as [a document](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library) says.
    
If the problems, especially the first one, are fixed, all loggers in `aiortc` package have a common prefix in thier names like `aiortc.xxx` and library users are able to configure the loggers by their own easily like below.
```python
import logging

my_own_handler = logging.StreamHandler()
my_own_handler.setLevel(logging.DEBUG)
my_own_formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")  # Users can use logging format preferable for them
my_own_handler.setFormatter(my_own_formatter)

aiortc_logger = logging.getLogger("aiortc")
aiortc_logger.setHandler(my_own_handler)
aiortc_logger.setLevel(logging.DEBUG)  # Users can specify the log level they want, specifically for aiortc package.
```

These points are described in https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library.

Thank you 😃

**Comment by jlaine** [comment]
Please feel free to work on a PR.

**Comment by jlaine** [comment]
This was fixed by @Laityned in #464, thanks!
